exporter/azuremonitorexporter/config.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azuremonitorexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for Azure Monitor
+type Config struct {
+	// squash ensures fields are correctly decoded in embedded struct.
+	configmodels.ExporterSettings `mapstructure:",squash"`
+	Endpoint                      string `mapstructure:"endpoint"`
+	InstrumentationKey            string `mapstructure:"instrumentation_key"`
+	MaxBatchSize                  int    `mapstructure:"maxbatchsize"`
+	MaxBatchIntervalInSeconds     int    `mapstructure:"maxbatchinterval_sec"`
+	Debug                         bool   `mapstructure:"debug"`
```

Comment:
Not sure about each exporter having a debug mode. I think it should be a global thing enabled with a CLI flag. Component should know if debug mode is enabled from the host.

Reply:
Probably true, however for the contrib repo I have to treat the actual collector host as a black box right? I do however think there is value in enabling additional output per receiver/exporter/etc...

Reply:
Removed the Debug mode. I am checking the zap.Logger to see if the debug level is enabled instead.